core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -76,6 +68,47 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
   checkMobileView(): boolean {
     return (this.windowRef.nativeWindow.innerWidth < 800);
   }
+
+  updateSelection(selection: string): void {
+    this.selection = selection;
+    this.setActiveTab(selection);
+  }
+
+  ngOnInit(): void {
+    let userInfoPromise = this.userService.getUserInfoAsync();
+    userInfoPromise.then(userInfo => {
+      const username = userInfo.getUsername();
+      if (username) {
+        this.isQuestionCoordinator = userInfo.isQuestionCoordinator();
+        this.isTranslationCoordinator = userInfo.isTranslationCoordinator();
+
+        if (this.isTranslationCoordinator) {
+          this.CONTRIBUTION_TYPES = [this.TAB_NAME_TRANSLATION_SUBMITTER,
+            this.TAB_NAME_TRANSLATION_REVIEWER];
+        }
+        if (this.isQuestionCoordinator) {
+          this.CONTRIBUTION_TYPES = [this.TAB_NAME_QUESTION_SUBMITTER,
+            this.TAB_NAME_QUESTION_REVIEWER];
+        }
+        if (this.isQuestionCoordinator && this.isTranslationCoordinator) {
+          this.CONTRIBUTION_TYPES = [this.TAB_NAME_TRANSLATION_SUBMITTER,
+            this.TAB_NAME_TRANSLATION_REVIEWER,
+            this.TAB_NAME_QUESTION_SUBMITTER,
+            this.TAB_NAME_QUESTION_REVIEWER];
+        }
+
+        this.updateSelection(this.CONTRIBUTION_TYPES[0]);
```

Comment:
PTAL at previous comment.

Reply:
Selected value is updated via dropdown and activeTab value is updated via higher level tabs.